.gitignore

```diff
@@ -11,3 +12,7 @@ packages/
 
 # Include when developing application packages.
 pubspec.lock
+
+
+# IDE files
+.idea/
```

Comment:
These should go in `.gitignore_global` – https://help.github.com/articles/ignoring-files/#create-a-global-gitignore